EasyPost.Tests/Fixture.cs

```diff
@@ -138,11 +143,11 @@ internal static class Parameters
         {
             internal static class Addresses
             {
-                internal static BetaFeatures.Parameters.Addresses.Create Create(Dictionary<string, object>? fixture)
+                internal static Create Create(Dictionary<string, object>? fixture)
```

Comment:
question: how is this getting away with just `Create`? Shouldn't it be `Addresses.Create`? 

question: Can we include the type of object to be more explicit? This is true of the whole PR, it would be easier to follow if done that way

question: Do we want to do singular vs plural to match services?

Reply:
1. My IDE cleaned up the imports during namespace adjustments. `Create` and `All` is all that's needed because this file imports the `EasyPost.Parameters.Address` namespace.

2. Because of 1 above, it shortened the namespace. We can certainly go back and make explicit namespaces (I'd be concerned the IDE would constantly be trying to auto-optimize them in the future, however)

3. I'll defer, noting that it would be a breaking change.

Reply:
We use singular in other libs. Since this is already a major bump, we should consider making it singular to match the service.

Reply:
As for item 1 I guess it's confusing because it's a fixture file with lots of different objects in one file, how does it know it's an Address.Create if it just says Create. 

For item 2, I'd prefer to be explicit and see if we can't override that so IDEs don't freak out. I'd be concerned with mixing namespaces for items in the same file

Reply:
1. If you concern was specifically for the Fixture file, already resolved.
2. Fine

Reply:
Resolved

Reply:
I'm still seeing a lot of items without the full namespace, by chance were some missed? There's a lot in here heh